libbeat/autodiscover/providers/jolokia/jolokia.go

```diff
@@ -92,6 +94,7 @@ func AutodiscoverBuilder(bus bus.Bus, uuid uuid.UUID, c *common.Config) (autodis
 		builders:  builders,
 		appenders: appenders,
 		discovery: discovery,
+		keystore:  keystore,
```

Comment:
you are not passing the keystore in jolokia events, should we add that too?

Reply:
👍